Fixes test failures on musl

Fixes errors
ptestresult.python3-dominate.tests/test_dom_tag.py::test___get_thread_context

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

diff --git a/tests/test_dom_tag.py b/tests/test_dom_tag.py
index 43ffecb..1126e54 100644
--- a/tests/test_dom_tag.py
+++ b/tests/test_dom_tag.py
@@ -18,6 +18,7 @@ def test___get_thread_context(monkeypatch):
     monkeypatch.setattr(sut, 'threading', threading)
 
     assert sut._get_thread_context() in [
+        1692341442, # Python >= 3.9, 32-bit
         -6805948436281256182, # Python >= 3.9
         3713141171098444831, # Python < 3.9
     ]
